docs/UserGuide.md

```diff
@@ -639,10 +642,10 @@ Parameters:
 
 * [`PROJECT_INDEX`](#index) The index of the project.
 * [`EVENT_INDEX`](#index) The index of an event.
-* [`DESCRIPTION`](#description) The description of an event.
-* [`DATE`](#date) The date of the event, or the starting date of the event if the event repeats every week.
-* [`TIME`](#time) The time of the event.
-* [`REPEAT_WEEKLY`](#repeat_weekly) Indicates whether the event repeats every week.
+* [`DESCRIPTION`](#description) The updated description of an event.
+* [`DATE`](#date) The updated date of the event, or the starting date of the event if the event repeats every week.
+* [`TIME`](#time) The updated time of the event.
+* [`REPEAT_WEEKLY`](#repeat_weekly) Updates indicating whether the event repeats every week.
```

Comment:
This line look a bit weird to me... I'm not sure what to change to tho...

Reply:
yeah i think should keep